plugins/woocommerce/src/Container.php

```diff
@@ -106,7 +108,7 @@ public function __construct() {
 	 *
 	 * @return mixed Entry.
 	 */
-	public function get( string $id ): object {
+	public function get( string $id ) {
```

Comment:
With [this addition](https://github.com/woocommerce/woocommerce/pull/39701/files#diff-97fb927371ee663e98afaf8b8134905a96d4b59c68f6a83c569ec42a6b468a08R265-R274), we can not get an array of objects instead of just a single object. This is actually one of the features of the Container class we're wrapping. With this return type in place, the array of object being returned causes an error, so I removed the return type.